code/game/objects/items/toys.dm

```diff
@@ -1034,7 +1034,6 @@
 	desc = "A compact ball of snow. Good for throwing at people."
 	icon = 'icons/obj/toy.dmi'
 	icon_state = "snowball"
-	throwforce = 12 //pelt your enemies to death with lumps of snow
```

Comment:
You should leave this as 12, but change the damtype to halloss or whichever it is in this codebase, like the rest of the holo-weapons, so that snowball fights tire people out.